_episodes_rmd/15-knitr-markdown.Rmd

```diff
@@ -52,7 +52,7 @@ data, you can just re-compile the report and get the new or corrected
 results (versus having to reconstruct figures, paste them into
 a Word document, and further hand-edit various detailed results).
 
-The key R package is `[knitr](http://yihui.name/knitr/)`. It allows you
+The key R package is [knitr](http://yihui.name/knitr/). It allows you
```

Comment:
Thanks for catching that! Ideally, to keep the package name properly stylized, this should be [`knitr`](http://yihui.name/knitr/) instead